src/Microsoft.Health.Dicom.Client/DicomWebClient.cs

```diff
@@ -203,21 +193,18 @@ public async Task<DicomWebResponse<IReadOnlyList<DicomDataset>>> RetrieveMetadat
         {
             await using (var stream = GetMemoryStream())
             {
-                await dicomFile.SaveAsync(stream);
+                await dicomFile.SaveAsync(stream).ConfigureAwait(false);
                 stream.Seek(0, SeekOrigin.Begin);
-                return await PostAsync(stream, studyInstanceUid, cancellationToken);
+                return await PostAsync(stream, studyInstanceUid, cancellationToken).ConfigureAwait(false);
             }
         }
 
         public async Task<DicomWebResponse> DeleteAsync(Uri requestUri, CancellationToken cancellationToken = default)
         {
             using var request = new HttpRequestMessage(HttpMethod.Delete, requestUri);
```

Comment:
nit: can we make it consistent (e.g., either change all to `using var request = ...` or change this to `using (var request = ...)`)?